### DISCUSSION TITLE
How to disable all /proc/net/snmp6 metrics?

### DISCUSSION THREAD

**Discussion by nicknaychov** [netdata/netdata#14839] [Q&A] [answered]
I do not use ip6 so I would like to disable those metrics.
In previous versions this used to work:
`[plugin:proc]
    /proc/net/snmp6 = no`

but not anymore.
Nothing in the docs. There should be a way to disable the whole family of snmp6 metrics.
The only way so far is to disable each metric for this plugin in netdata.conf, however those metrics are around 20.

Thank you

**Comment by ilyam8** [comment, marked as answer]
Hi, @nicknaychov. There is no option to disable collecting metrics from `/proc/net/snmp6`. If your system has IPv6 enabled Netdata will collect IPv6 metrics.

A workaround is to change `[plugin:proc:/proc/net/snmp6] filename to monitor` to a file that doesn't exist.

**Comment by Bilalxxx12** [comment]
Hi
